### PR TITLE
`go build` / `go install` should generate `tofu` binary

### LINKED ISSUE #583: `go build` / `go install` should build package as `tofu` and not `opentofu`
Currently when running 
`go install` or `go build` the generated package is `opentofu`, while it should be `tofu`

### PR DESCRIPTION
**Pull request by eranelbaz**
Resolves #583
Now we run the command under the `cmd/tofu` folder

## Target Release

1.6.0

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

**Comment by pdecat** [comment]
Awesome! 

Maybe also change .gitignore at the same time: https://github.com/opentofu/opentofu/blob/main/.gitignore#L21C6-L21C6

**Comment by eranelbaz** [comment]
@pdecat 81a74b89

**Review by cube2222 — APPROVED** [review]
LGTM!

File: scripts/build.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ SOURCE="${BASH_SOURCE[0]}"
 while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
 DIR="$( cd -P "$( dirname "$SOURCE" )/.." && pwd )"
 
+echo $DIR
```
Comment (serdardalgic) [inline scripts/build.sh:13]: Not that I'm against it, is this intended or a forgotten `echo $DIR`? 🤔

Comment (eranelbaz) [reply scripts/build.sh]: Forgotten 😓 8bef10ec(remove echo)

**Pull request merged**